pages/wi/partner/[slug].js

```diff
@@ -1,17 +1,19 @@
 import React from 'react';
+import { useRouter } from 'next/router';
 import styled from 'styled-components';
 import { useQuery } from '@apollo/react-hooks';
 import { gql } from 'apollo-boost';
 
-import HeroSection from '../../components/PartnerDetail/HeroSection';
-import MainLogoSection from '../../components/PartnerDetail/MainLogoSection';
-import AboutGoalsSection from '../../components/PartnerDetail/AboutGoalsSection';
-import PresentationsJobsSection from '../../components/PartnerDetail/PresentationsJobsSection';
+import HeroSection from '../../../components/PartnerDetail/HeroSection';
```

Comment:
There is a way we can reference from the root rather than `../` hell. I forget how, maybe @saragibby knows. Future enhancement we might want to add to help clean up things.